fix(curriculum): CSS relative and absolute units quiz (#56552)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/quiz-css-relative-and-absolute-units/66ed9009f45ce3ece4053eb7.md

#### --text--

If the body text is 16px, how much is 1em in most child elements?

#### --distractors--

18px

---

12px

---

It depends on the parent’s font size

#### --answer--

> 16px

Reviewer: do not remove empty lines please
```suggestion

16px
```

#### --text--

> If the body text is 16px, how much is 1em in most child elements?

Reviewer 2: ```suggestion
If the body text is `16px`, how much is `1em` in most child elements?
```

#### --distractors--

> 18px

Reviewer 2: ```suggestion
`18px`
```

---

> 12px

Reviewer 2: ```suggestion
`12px`
```

---

> It depends on the parent’s font size

Reviewer 3: ```suggestion
It depends on the parent's font size
```

#### --answer--

> 16px

Reviewer 2: ```suggestion
`16px`
```

### --question--

#### --text--

> What does 100vw cover in terms of screen width?

Reviewer 2: ```suggestion
What does `100vw` cover in terms of screen width?
```

#### --distractors--

100% of the screen width answer

---

100% of the viewport height

---

> 100% of the parent’s width

Reviewer 3: ```suggestion
100% of the parent's width
```

#### --answer--

100% of the viewport width

### --question--

#### --text--

> If a container is 1000px wide, how wide is 5% of it?

Reviewer 2: ```suggestion
If a container is `1000px` wide, how wide is 5% of it?
```

#### --distractors--

> 100px

Reviewer 2: ```suggestion
`100px`
```

---

> 500px

Reviewer 2: ```suggestion
`500px`
```

---

> 5px

Reviewer 2: ```suggestion
`5px`
```

#### --answer--

> 50px

Reviewer 2: ```suggestion
`50px`
```

### --question--

#### --text--

> If the root font size is 20px, what is 1rem equal to?

Reviewer 2: ```suggestion
If the root font size is `20px`, what is `1rem` equal to?
```

#### --distractors--

> 18px

Reviewer 2: ```suggestion
`18px`
```

---

> 16px

Reviewer 2: ```suggestion
`16px`
```

---

> 10px

Reviewer 2: ```suggestion
`10px`
```

#### --answer--

> 20px

Reviewer 2: ```suggestion
`20px`
```

### --question--

#### --text--

> What can the calc() function do in CSS?

Reviewer 2: ```suggestion
What can the `calc()` function do in CSS?
```

#### --distractors--

Modify colors dynamically

---

Set a fixed width

---

Only work with percentages

#### --answer--

Add or subtract different units

### --question--

#### --text--

> How many points (pt) are in 1 pixel (px)?

Reviewer 2: ```suggestion
Approximately how many points (`pt`) are in 1 pixel (`px`)?
```

#### --distractors--

> About 1.33pt

Reviewer 2: ```suggestion
`1.33pt`
```

---

> About 1pt

Reviewer 2: ```suggestion
`1pt`
```

---

> About 2pt

Reviewer 2: ```suggestion
`2pt`
```

#### --answer--

> About 0.75pt

Reviewer 2: ```suggestion
`0.75pt`
```

### --question--

#### --text--

> If the screen height is 800px, what would 50vh be?

Reviewer 2: ```suggestion
If the screen height is `800px`, what would `50vh` be?
```

#### --distractors--

> 50px

Reviewer 2: ```suggestion
`50px`
```

---

> 200px

Reviewer 2: ```suggestion
`200px`
```

---

> 100px

Reviewer 2: ```suggestion
`100px`
```

#### --answer--

> 400px

Reviewer 2: ```suggestion
`400px`
```

### --question--

#### --text--

> What does setting an element’s width to 100% mean?

Reviewer 2: ```suggestion
What does setting an element’s width to `100%` mean?
```

Reviewer 3: ```suggestion
What does setting an element's width to `100%` mean?
```

#### --distractors--

It takes up the full width of the document

---

It takes up the full width of the screen

---

> It takes up 100px

Reviewer 2: ```suggestion
It takes up `100px`
```

#### --answer--

It takes up the full width of its parent

### --question--

#### --text--

> What happens with calc(10px + 20%) in CSS?

Reviewer 2: ```suggestion
What happens with `calc(10px + 20%)` in CSS?
```

#### --distractors--

> Adds 10px to 10% of the document width

Reviewer 2: ```suggestion
Adds `10px` to `10%` of the document width
```

---

> Always equals 30px

Reviewer 2: ```suggestion
Always equals `30px`
```

---

> Adds 10px to 20% of the screen width

Reviewer 2: ```suggestion
Adds `10px` to `20%` of the screen width
```

#### --answer--

> Adds 10px and 20% of the parent’s width

Reviewer 2: ```suggestion
Adds `10px` to `20%` of the parent’s width
```

Reviewer 3: ```suggestion
Adds `10px` and `20%` of the parent's width
```

### --question--

#### --text--

> If the parent element’s text size is 20px, how big is 1.5em?

Reviewer 2: ```suggestion
If the parent element’s text size is `20px`, how big is `1.5em`?
```

Reviewer 3: ```suggestion
If the parent element's text size is `20px`, how big is `1.5em`?
```

#### --distractors--

> 25px

Reviewer 2: ```suggestion
`25px`
```

---

> 15px

Reviewer 2: ```suggestion
`15px`
```

---

> 20px

Reviewer 2: ```suggestion
`20px`
```

#### --answer--

> 30px

Reviewer 2: ```suggestion
`30px`
```

### --question--

#### --text--

> With a root font size of 16px, how big is 10rem?

Reviewer 2: ```suggestion
With a root font size of `16px`, how big is `10rem`?
```

#### --distractors--

> 140px

Reviewer 2: ```suggestion
`140px`
```

---

> 10px

Reviewer 2: ```suggestion
`10px`
```

---

> 100px

Reviewer 2: ```suggestion
`100px`
```

#### --answer--

> 160px

Reviewer 2: ```suggestion
`160px`
```

### --question--

#### --text--

> Why might you use pt instead of px in design?

Reviewer 2: ```suggestion
Why might you use `pt` instead of `px` in design?
```

#### --distractors--

Points are scalable

---

Points are better for responsive layouts

---

Points are easier to manage on screens

#### --answer--

Points are for print design

### --question--

#### --text--

> How does calc(10vh + 50px) work for height?

Reviewer 2: ```suggestion
How does `calc(10vh + 50px)` work for height?
```

#### --distractors--

> Adds 50px to 10% of the screen height

Reviewer 2: ```suggestion
Adds `50px` to `10%` of the screen height
```

---

> Adds 10px to 50% of the parent’s height

Reviewer 2: ```suggestion
Adds `10px` to `50%` of the parent’s height
```

Reviewer 3: ```suggestion
Adds `10px` to `50%` of the parent's height
```

---

> Adds 10vh to 50% of the document

Reviewer 2: ```suggestion
Adds `10vh` to `50%` of the document
```

#### --answer--

> Adds 50px to 10% of the screen height

Reviewer 2: ```suggestion
Adds `50px` to `10%` of the screen height
```

### --question--

#### --text--

> How much screen space does 80vw take?

Reviewer 2: ```suggestion
How much screen space does `80vw` take?
```

#### --distractors--

> 80% of the parent’s width

Reviewer 3: ```suggestion
80% of the parent's width
```

---

100% of the viewport height

---

> 80px

Reviewer 2: ```suggestion
`80px`
```

#### --answer--

80% of the screen width

### --question--

#### --text--

> If the root font size is 18px, how big is 2rem?

Reviewer 2: ```suggestion
If the root font size is `18px`, how big is `2rem`?
```

#### --distractors--

> 24px

Reviewer 2: ```suggestion
`24px`
```

---

> 18px

Reviewer 2: ```suggestion
`18px`
```

---

> 20px

Reviewer 2: ```suggestion
`20px`
```

#### --answer--

> 36px

Reviewer 2: ```suggestion
`36px`
```

### --question--

#### --text--

> What does calc(50% + 20px) do for width?

Reviewer 2: ```suggestion
What does `calc(50% + 20px)` do for width?
```

#### --distractors--

> Adds 50% of the screen width plus 20px

Reviewer 2: ```suggestion
Adds `50%` of the screen width plus `20px`
```

---

> Adds 50px to the screen width

Reviewer 2: ```suggestion
Adds `50px` to the screen width
```

---

> Adds 50% of the height plus 20px

Reviewer 2: ```suggestion
Adds `50%` of the height plus `20px`
```

#### --answer--

> Adds 50% of the parent’s width plus 20px

Reviewer 2: ```suggestion
Adds `50%` of the parent’s width plus `20px`
```

Reviewer 3: ```suggestion
Adds `50%` of the parent's width plus `20px`
```

### --question--

#### --text--

> What’s the key difference between rem and em in CSS?

Reviewer 2: ```suggestion
What’s the key difference between `rem` and `em` in CSS?
```

Reviewer 3: ```suggestion
What's the key difference between `rem` and `em` in CSS?
```

#### --distractors--

> rem is only used for fonts

Reviewer 2: ```suggestion
`rem` is only used for fonts
```

---

> rem is for width, em is for fonts

Reviewer 2: ```suggestion
`rem` is for width, `em` is for fonts
```

---

> rem is always smaller than em

Reviewer 2: ```suggestion
`rem` is always smaller than `em`
```

#### --answer--

> rem is based on the root font size, em is based on the parent’s font size

Reviewer 2: ```suggestion
`rem` is based on the root font size, `em` is based on the parent’s font size
```

Reviewer 3: ```suggestion
`rem` is based on the root font size, `em` is based on the parent's font size
```

### --question--

#### --text--

> How does calc(100px + 10%) work for width?

Reviewer 2: ```suggestion
How does `calc(100px + 10%)` work for width?
```

#### --distractors--

> Adds 10% of the screen width to 100px

Reviewer 2: ```suggestion
Adds `10%` of the screen width to `100px`
```

---

> Adds 10% of the height to 100px

Reviewer 2: ```suggestion
Adds `10%` of the height to `100px`
```

---

> Adds 100px to 10% of the document

Reviewer 2: ```suggestion
Adds `100px` to `10%` of the document
```

#### --answer--

> Adds 10% of the parent’s width to 100px

Reviewer 2: ```suggestion
Adds `10%` of the parent’s width to `100px`
```

Reviewer 3: ```suggestion
Adds `10%` of the parent's width to `100px`
```

### --question--

#### --text--

> What is the pixel value of 1 point (pt) in CSS?

Reviewer 2: ```suggestion
What is the pixel value of 1 point (`1pt`) in CSS?
```

#### --distractors--

> 1.33px

Reviewer 2: ```suggestion
`1.33px`
```

---

> 2px

Reviewer 2: ```suggestion
`2px`
```

---

> 1px

Reviewer 2: ```suggestion
`1px`
```

> What is the pixel value of 1 point (`1pt`) in CSS?

Reviewer: this question is still repeated, I think you should change it with a different topic

#### --answer--

> 0.75px

Reviewer 2: ```suggestion
`0.75px`
```

### --question--

#### --text--

> What happens when you use calc(30vh - 10px) for height?

Reviewer 2: ```suggestion
What happens when you use `calc(30vh - 10px)` for height?
```

#### --distractors--

> Subtracts 10px from the parent’s height

Reviewer 2: ```suggestion
Subtracts `10px` from the parent’s height
```

Reviewer 3: ```suggestion
Subtracts `10px` from the parent's height
```

---

> Subtracts 30px from the screen height

Reviewer 2: ```suggestion
Subtracts `30px` from the screen height
```

---

> Adds 30% of the parent’s height minus 10px

Reviewer 2: ```suggestion
Adds `30%` of the parent’s height minus `10px`
```

Reviewer 3: ```suggestion
Adds `30%` of the parent's height minus `10px`
```

#### --answer--

> Subtracts 10px from 30% of the screen height

Reviewer 2: ```suggestion
Subtracts `10px` from `30%` of the screen height
```

#### --text--

> What does setting an element's width to auto do in CSS?

Reviewer: It's not a bad question, but `auto` is not mentioned in the topics for this quiz. WHat does @naomi-lgbt say?

Author: @naomi-lgbt  is it OK..??

Reviewer 2: Should be fine.

Author: When it will be merged...??

Reviewer: The rules below requires that there are no unresolved conversations and that the requests of changes are satisfied
